CHANGELOG.md

```diff
@@ -72,6 +72,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add buffered lookahead for Jackson ([#338](https://github.com/opensearch-project/opensearch-java/pull/338))
 - Add support for headers and sort parameters in cat requests ([#413](https://github.com/opensearch-project/opensearch-java/pull/413))
 - Add support for data stream operations ([#419](https://github.com/opensearch-project/opensearch-java/pull/419))
+- Add missing minScore, postFilter, searchAfter, sort, trackScores in the MultisearchBody ([#516](https://github.com/opensearch-project/opensearch-java/pull/516))
```

Comment:
This changelog entry needs revert I believe

Reply:
I agree, it is probably useless to modify changelog of a released version